tests/test_wordpress_api.py

```diff
@@ -95,6 +95,7 @@ def test_getting_24_articles_from_page_3(self, get):
             + "&categories="
             + "&exclude="
             + "&author="
+            + "&_embed"
```

Comment:
Hm,

this one https://github.com/canonical-web-and-design/canonicalwebteam.blog/pull/55 should simplify testing quite a bit :)

Maybe you want to rebase when its merged?